Configuration/PyReleaseValidation/python/relval_standard.py

```diff
@@ -278,6 +278,7 @@
 workflows[1325.1] = ['', ['TTbar_13','DIGIUP15','RECOUP15_trackingOnly','HARVESTUP15_trackingOnly']]
 workflows[1325.2] = ['', ['TTbar_13','DIGIUP15','RECOUP15_trackingLowPU','HARVESTUP15']]
 workflows[1325.3] = ['', ['TTbar_13','DIGIUP15','RECOUP15_trackingOnlyLowPU','HARVESTUP15_trackingOnly']]
+workflows[1325.4] = ['', ['TTbar_13','DIGIUP15','RECOUP15_HIPM','HARVESTUP15']]
```

Comment:
Ciao Slava,
I guess I do not understand your comment. This is 81X and I thought the rereco will be performed in 80X, the 2016 data-taking release.
Since 81X is not meant to be a 2016 p-p data taking release we should safely revert all VFP30 related changes for good, while keeping the possibility of using the mitigation via Era.
Also, within a release, AFAIK, we have no way to enforce IOV-based (on prompt-reco eras, i.e., runs) reconstruction, right?
I guess I really miss the point of your comment.